msteams-platform/bots/calls-and-meetings/requirements-considerations-application-hosted-media-bots.md

```diff
@@ -19,7 +19,7 @@ An application-hosted media bot requires the [`Microsoft.Graph.Communications.Ca
 
 An application-hosted media bot requires the following:
 
-* The bot must be developed in C# and the standard .NET Framework and deployed in Microsoft Azure. You can't use C++ or Node.js APIs to access real-time media and .NET Core isn't supported for an application-hosted media bot.
+* The bot must be developed in C# and the standard .NET Framework and deployed in Microsoft Azure. You can't use C++ or Node.js APIs to access real-time media and .NET Core is supported for an application-hosted media bot and sdk supports dotnet 6.
```

Comment:
Is donet 6 the correct usage. Should we write it as .NET 6.0 ?

Reply:
I have changed it to .Net 6.0.